runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -70,20 +70,31 @@ default int getIdleNotificationDelay()
 
 	@ConfigItem(
 		keyName = "hitpoints",
-		name = "Hitpoints Notification Threshold",
-		description = "The amount of hitpoints to send a notification at. A value of 0 will disable notification.",
+		name = "Low Hitpoints Threshold",
+		description = "Send a notification when you fall below this amount of hitpoints. A value of 0 will disable notification.",
 		position = 4
 	)
 	default int getHitpointsThreshold()
 	{
 		return 0;
 	}
 
+	@ConfigItem(
+			keyName = "hp",
+			name = "High Hitpoints Threshold",
+			description = "Send a notification when you are above this amount of hitpoints. A value of 0 will disable notification.",
+			position = 5
+	)
+	default int getHPThreshold()
```

Comment:
You should probably make enum to switch between low and high HP warnings and use 1 field for them , probably also with adding option OFF.